### PR TITLE
8294321: Fix typos in files under test/jdk/java, test/jdk/jdk, test/jdk/jni

### PR DESCRIPTION
**Pull request by jaikiran**
Can I please get a review of this change which fixes the typos in the test packages? @mernst submitted this as a separate PR https://github.com/openjdk/jdk/pull/10029 but given the number of areas and files that other PR touches, the progress has been stalled.

I'll raise separate PRs in the other remaining areas from that other PR.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8294321](https://bugs.openjdk.org/browse/JDK-8294321): Fix typos in files under test/jdk/java, test/jdk/jdk, test/jdk/jni

### Reviewers
 * [Alan Bateman](https://openjdk.org/census#alanb) (@AlanBateman - **Reviewer**) ⚠️ Review applies to [85669caf](https://git.openjdk.org/jdk/pull/11385/files/85669caf0fedabf729a178e1b8c88fdf88fc2ab9)
 * [Lance Andersen](https://openjdk.org/census#lancea) (@LanceAndersen - **Reviewer**)
 * [Daniel Fuchs](https://openjdk.org/census#dfuchs) (@dfuch - **Reviewer**)

### Contributors
 * Michael Ernst `<mernst@openjdk.org>`

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/11385/head:pull/11385` \
`$ git checkout pull/11385`

Update a local copy of the PR: \
`$ git checkout pull/11385` \
`$ git pull https://git.openjdk.org/jdk pull/11385/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 11385`

View PR using the GUI difftool: \
`$ git pr show -t 11385`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/11385.diff">https://git.openjdk.org/jdk/pull/11385.diff</a>

</details>

### CODE REVIEW

**Review by AlanBateman — APPROVED** [review]

File: test/jdk/java/net/httpclient/reactivestreams-tck/org/reactivestreams/tck/WithHelperPublisher.java [hunk 1]
```diff
@@ -64,7 +64,7 @@
    * By default an <b>asynchronously signalling Publisher</b> is provided, which will use {@link #createElement(int)}
    * to generate elements type your Subscriber is able to consume.
    * <p>
-   * Sometimes you may want to implement your own custom custom helper Publisher - to validate behaviour of a Subscriber
+   * Sometimes you may want to implement your own custom helper Publisher - to validate behaviour of a Subscriber
```
[the remark this answers is not in the document]
Comment (jaikiran) [reply test/jdk/java/net/httpclient/reactivestreams-tck/org/reactivestreams/tck/WithHelperPublisher.java]: Hello Alan, I have now updated this PR to remove changes to this file.

**Review by LanceAndersen — APPROVED** [review]

**Review by dfuch — APPROVED** [review]

**Comment by jaikiran** [comment]
Thank you everyone for the reviews.

**Pull request closed without merging**